src/assets/footer-navigation.json

```diff
@@ -176,7 +176,7 @@
         },
         {
           "label": "Support",
-          "absUrl": "mailto:support@topcoder.com"
+          "url": "mailto:support@topcoder.com"
```

Comment:
Is this what's in the spec?  I think this should actually go to the support page rather than just open the email.

https://platform-ui.topcoder.com/support

Reply:
@brooketopcoder yes, it's in the jira ticket as well in lucidchart
https://topcoder.atlassian.net/browse/PROD-3142
![image](https://user-images.githubusercontent.com/2527433/204368681-1a301aaf-5fd9-410c-89aa-b725d7ad23f2.png)

Reply:
Okay--I will ask Oahn to update the specs. Thanks!